apps/prairielearn/src/pages/administratorCourses/administratorCourses.sql

```diff
@@ -88,28 +18,11 @@ WITH
       JOIN institutions AS i ON (i.id = c.institution_id)
     WHERE
       c.deleted_at IS NULL
-  ),
-  select_institutions AS (
-    SELECT
-      coalesce(
-        jsonb_agg(
-          i
-          ORDER BY
-            i.short_name
-        ),
-        '[]'::jsonb
-      ) AS institutions
-    FROM
-      institutions AS i
   )
 SELECT
-  course_requests,
-  courses,
-  institutions
+  courses
 FROM
-  select_course_requests,
-  select_courses,
-  select_institutions;
+  select_courses;
```

Comment:
It looks like this whole CTE is unnecessary now, as is pre-aggregating the courses into a JSON array. This could be pretty dramatically simplified now. Remember that you'll need to update the Zod schema to account for any changes you make to simplify things.

Reply:
This has now been simplified in commit [46a7046](https://github.com/PrairieLearn/PrairieLearn/pull/9238/commits/46a7046a6311dae7797f62cb4227fcbf1a3ead90).